src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+## How data is exported by Augur for visualisation in Auspice
+
+This page is intended to describe which data underpins the different parts of an Auspice visualisation, and how this data can be produced via Augur. It is not a comprehensive guide to how to generate individual pieces of data nor how to interpret them in Auspice. 
+
+### Auspice components
+
+It’s helpful to start in auspice and then work backwards to augur. The following (annotated) screenshot shows the components of a typical visualisation, which is based on a dataset JSON (sometimes called an Auspice JSON). 
+
+Generally, data is defined on individual tree nodes in this dataset (via `node.node_attrs: Dictionary` or `node.branch_attrs: Dictionary` ) or within `meta: Dictionary` . We will refer to these below, however the best place to really understand the data structures is  [the JSON schema](https://github.com/nextstrain/augur/blob/master/augur/data/schema-export-v2.json).  
+
+This guide is based on Auspice version 2.37.3. We will endeavour to keep this up-to-date however if you are using a newer version of auspice it may pay to check the [changelog](https://github.com/nextstrain/auspice/blob/master/CHANGELOG.md) for any salient changes.
+
+![](../../images/auspice-components.png)
+
+**[a1] Colorings**
+
+The available colorings are defined by  `meta.colorings: Array<dict>`, where each colouring specifies a key which accesses the relevant data in the `node_attrs` [1]. As well as a key, a title can be specified (which is what the user will see), as well as information about the scale used.
+
+**[a2] sidebar data filtering**
+
+Currently the  `node_attrs` available for filtering are the union of those defined in `meta.filters: Array<string>` , strain names (`node.name: String`) and mutations (`branch_attrs.mutations` ).
+
+> We will hopefully change this shortly to include all data defined in a `node.node_attrs`.
+
+**[a3] Temporal display of the tree**
+
+If `num_date` is set on _all_ nodes (including internal nodes) then the “TIME” branch metric and the “Date Range” selector is available. Note that to colour the tree by this a corresponding entry in the colorings is required. Conversely, for “DIVERGENCE” then `div` (cumulative) must be set on all nodes in the tree. 
```

Comment:
If converting to rST for other reasons (see other comments), consider using the `guilabel` role for things like "TIME" and "Date Range", e.g.

```rst
then the :guilabel:`TIME` branch metric and the :guilabel:`Date Range` selector is available.
```

You can see an example of how our theme (currently) renders those in [the fourth ¶ of this demo doc section]( https://docs.nextstrain.org/projects/sphinx-theme/en/stable/demo/demo.html#inline-markup).

Reply:
I did this in [af41123](https://github.com/nextstrain/docs.nextstrain.org/pull/113/commits/af4112391b0e1698724df71cc43c6220a25c90c0).